CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Change Log
+
+## [1.3.12] - 2019-29-10
+### Changed
+
+- Reduce response time for NPM commands.
+- Remove unnecessary console statements and double logs.
+
+## [1.3.11] - 2019-29-10
+### Fixed
+
+- Fix problem with wait ticks. They do not lock the system now.
+
```

Comment:
@anacierdem what went into 1.3.10 ?

Reply:
I skipped that one due to a mistake :) I'm a little impatient when it comes to releases, trying to improve myself on that regard :D